_includes/footer.html

```diff
@@ -53,6 +53,18 @@
             <span class="username">{{ site.twitter_username }}</span>
           </a>
         </li>{% endif %}
+
+        {% if site.mastodon_username %}<li>
+          <a href="{{ site.mastodon_link }}{{ site.mastodon_username }}">
```

Comment:
```suggestion
          <a href="{{ site.mastodon_link }}{{ site.mastodon_username }}" rel="me">
```

how about adding a `rel="me"`?

https://joinmastodon.org/verification